### PR TITLE
refactor(sensors_monitor): ros2 port

### PR DESCRIPTION
**Pull request by reinzor**

### CODE REVIEW

File: diagnostic_common_diagnostics/diagnostic_common_diagnostics/sensors_monitor.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+
+if __name__ == '__main__':
+    rclpy.init()
+    hostname = socket.gethostname()
+    hostname_clean = hostname.translate(hostname.maketrans('-', '_'))
+    node = rclpy.create_node('sensors_monitor_%s' % hostname_clean)
+
+    monitor = SensorsMonitor(node, hostname)
+    rclpy.spin(node)
```
Comment (Timple) [inline diagnostic_common_diagnostics/diagnostic_common_diagnostics/sensors_monitor.py]: This node crashes on exit, `KeyboardInterrupt` should be caught.

**Comment by reinzor** [comment]
ping @ct2034

**Comment by reinzor** [comment]
@ct2034 , can we merge this?

**Comment by reinzor** [comment]
ping @ct2034

**Comment by MCFurry** [comment]
@ct2034 can this be merged? Would love to have this node natively available again!

**Comment by ct2034** [comment]
Yes, after #362 and #363

**Comment by MCFurry** [comment]
> Yes, after #362 and #363

Cool!

**Comment by ct2034** [comment]
merged manually in e67a69c

**Pull request closed without merging**